sw/device/rom_exts/manifest.md

```diff
@@ -184,13 +184,35 @@ Notes:
     This is used when signing and validating the image. This happens in the
     Mask ROM, as well as during firmware update.
 
-1.  **Usage Constraints** This is a 32-bit enumeration value which denotes what
-    kinds of devices the ROM_EXT image may be used on.
-
-    This is used by the Mask ROM to validate the image is running on the right
-    kind of hardware.
-
-    **Open Q** What possible values should this have?
+1.  **Usage Constraints** This denotes which pieces of the OpenTitan Device
+    Identifier should also be added to the digest of this image. The OpenTitan
+    Device Identifier is a 256-bit value provisioned into the device's OTP
+    memory early in manufacturing.
+
+    This allows the ROM_EXT author to connect a ROM_EXT to a single device or a
+    group of devices, such that the ROM_EXT will not boot on devices not in that
+    group. This can use the Silicon Creator identifier, the product identifier,
+    the individual device identifier, or any data from the SKI-specific device
+    identifier.
+
+    This is a 32-bit unsigned numeric value.
+
+    Each bit in the usage constraint value denotes a byte in the 256bit Device
+    Identifier. If that bit is set, then that byte in the should be added to the
+    end of the digest, otherwise a zero-byte should be added to the end of the
```

Comment:
I think using the entire 256bit DI with bytes masked out isn't sensitive to reordering. It's probably not a big deal, but @moidx might have opinions.

Reply:
sgtm. I think it is safe to do it the way it is described here. I'd replace the zero-byte for a constant with high hamming distance.